src/OpenApi/test/Integration/snapshots/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=forms.verified.txt

```diff
@@ -364,6 +283,61 @@
       }
     }
   },
+  "components": {
+    "schemas": {
+      "boolean": {
+        "type": "boolean"
+      },
+      "DateTime": {
+        "type": "string",
+        "format": "date-time"
+      },
+      "IFormFile": {
+        "type": "string",
+        "format": "binary"
+      },
+      "IFormFileCollection": {
+        "type": "array",
+        "items": {
+          "$ref": "#/components/schemas/IFormFile"
+        }
+      },
```

Comment:
Does this potentially bleed out some ".NET-ness" where more generic names could be used (`form-file` or whatever)?

Reply:
Potentially, if applicable, I think I might prefer `FormFile` over `form-file` since it is a bit more homogenous with remaining conventions and more closely aligns with valid identifier names in most programming languages. There's probably some implementation bleed around disclosing that this is an interface.